src/bin/hq.rs

```diff
@@ -124,42 +122,45 @@ enum ServerCommand {
     Info(ServerInfoOpts),
 }
 
-#[derive(Clap)]
-#[clap(setting = clap::AppSettings::ColoredHelp)]
-pub struct WaitOpts {
-    selector: JobSelectorArg,
-}
-
-enum WorkerSelectorArg {
+enum SelectorArg {
     All,
-    Id(WorkerId),
+    Last,
+    Id(IntArray),
 }
 
-impl FromStr for WorkerSelectorArg {
+impl FromStr for SelectorArg {
     type Err = anyhow::Error;
 
     fn from_str(s: &str) -> Result<Self, Self::Err> {
         match s {
-            "all" => Ok(WorkerSelectorArg::All),
-            _ => Ok(WorkerSelectorArg::Id(FromStr::from_str(s)?)),
+            "last" => Ok(SelectorArg::Last),
+            "all" => Ok(SelectorArg::All),
+            _ => Ok(SelectorArg::Id(IntArray::from_str(s)?)),
         }
     }
 }
 
-impl From<WorkerSelectorArg> for WorkerSelector {
-    fn from(selector: WorkerSelectorArg) -> Self {
-        match selector {
-            WorkerSelectorArg::Id(job_id) => WorkerSelector::Specific(vec![job_id]),
-            WorkerSelectorArg::All => WorkerSelector::All,
+impl From<SelectorArg> for Selector {
+    fn from(selector_arg: SelectorArg) -> Self {
+        match selector_arg {
+            SelectorArg::Id(array) => Selector::Specific(array),
+            SelectorArg::Last => Selector::LastN(1),
+            SelectorArg::All => Selector::All,
         }
     }
 }
 
+#[derive(Clap)]
+#[clap(setting = clap::AppSettings::ColoredHelp)]
+pub struct WaitOpts {
+    selector_arg: SelectorArg,
```

Comment:
Please add a doc comment (`///`) to there arguments so that users know what they represent.